app/blueprints/fund_builder/templates/login.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+
+{# Override main width to two thirds #}
+{% set mainClasses = "govuk-!-width-two-thirds" %}
+
+{% block content %}
+  <div class="govuk-width-container">
+    <h1 class="govuk-heading-l">Sign in to use FAB</h1>
```

Comment:
Unsure whether we should have the acronym here or spell out the "service" name in full - mild content question?

Reply:
I believe design is looking at the login page and will come back with a re-designed version including potentially this change you've described, which can be captured in a future PR if you're happy with that